picky-asn1-x509/src/pkcs12.rs

```diff
@@ -0,0 +1,233 @@
+//! This module implements PKCS#12 specification defined in [RFC7292](https://tools.ietf.org/html/rfc7292).
```

Comment:
I believe the `mod.rs` convention is used everywhere else in picky codebase, so I suggest we stick to it.

As an aside, I personally prefer the `mod.rs` convention so that all files related to a given module are all in the same folder, but I’m willing to be convinced.

Reply:
Specifically in `picky-asn1-x509` there we only single example of module subfolder - `pkcs7` and it does not use `mod.rs` convention, so I used same approach in `pkcs12` but I completely agree with you, mod.rs is definitely _superior_ 😄 

Well, then while I am at it, I'll restructure `pkcs7` too

Reply:
Oh, I see! I probably missed that in a previous review, thank you!